app/build.gradle

```diff
@@ -50,5 +51,5 @@ play {
 }
 
 dependencies {
-    implementation project(':plugin-commons')
+    implementation 'com.github.vanilla-music:vanilla-music-plugin-commons:v1.0.0'
```

Comment:
It's 1.0.0, without "v". Sorry, all other plugins follow same scheme, can't change it for one library.